boards/lilygo/ttgo_t8s3/doc/index.rst

```diff
@@ -0,0 +1,247 @@
+.. _ttgo_t8s3:
+
+Lilygo TTGO T8-S3
+#################
+
+Overview
+********
+
+Lilygo TTGO T8-S3 is an IoT mini development board based on the
+Espressif ESP32-S3 WiFi/Bluetooth dual-mode chip.
+
+It features the following integrated components:
+
+- ESP32-S3 chip (240MHz dual core, Bluetooth LE, Wi-Fi)
+- on board antenna and IPEX connector
+- USB-C connector for power and communication
+- MX 1.25mm 2-pin battery connector
+- JST SH 1.0mm 4-pin UART connector
+- SD card slot
+
+.. figure:: img/ttgo_t8s3.webp
+   :align: center
+   :alt: TTGO T8-S3
+
+   Lilygo TTGO T8-S3
+
```

Comment:
and this can be removed too as the zephyr-board directive will take care of including the image automatically :)